.github/workflows/full-downstream.yml

```
@@ -31,23 +31,23 @@ jobs:
     name: Full downstream build
     steps:
       - name: Clean Disk Space
-        uses: kiegroup/kogito-pipelines/.ci/actions/ubuntu-disk-space@1.13.x
+        uses: kiegroup/.ci/actions/ubuntu-disk-space@main
```

It seems you removed `kogito-pipelines` instead of replacing it with `kie-ci`
```suggestion
        uses: kiegroup/kie-ci/.ci/actions/ubuntu-disk-space@main
```

---

the same applies on all references here